accounting/bank/setup/bank_accounts.rst

```diff
@@ -0,0 +1,95 @@
+=============
+Bank Accounts
+=============
+
+You can manage as many **Bank Accounts** as needed on your database. Configuring them well allows
+you to make sure that all your banking data is up to date and ready for the reconciliation with your
+Journal Entries.
+
+In Odoo Accounting, each Bank Account is configured to have a dedicated *Journal* which is
+configured to post all entries in a dedicated *Account*.
+
+.. note:: Whenever you add a Bank Account, a dedicated journal and a dedicated account are
+   automatically created and configured.
+
+Every **Bank Journal** is displayed by default on the **Accounting Overview** in the form of a
+convenient card. It includes action buttons that are displayed when appropriate.
+
+.. image:: media/bank_accounts_card.png
+   :align: center
+   :alt: Bank Journals Cards are displayed on the Accounting Overview in Odoo Accounting
+
+.. _bank_accounts_add:
+
+Add a new Bank Account
+======================
+
+You can either connect your bank account to your Odoo database, or configure your bank account
+manually and :doc:`upload the bank statements manually <../feeds/bank_statements>`.
+
+Bank Synchronization
+--------------------
+
+Connect your bank account to your database, and have your bank statements synced automatically.
+
+To synchronize a new bank account, go to :menuselection:`Accounting --> Configuration`, click on
+*Add a Bank Account*, then find your bank in the list, click on *Connect*, and follow the
+instructions on-screen.
+
+.. note::
+   :doc:`Click here <../../bank/feeds/bank_synchronization>` for more information about this bank
+   synchronization.
+
+.. image:: media/bank_accounts_connect.png
+   :align: center
+   :alt: Select a bank institution in the list and connect it to Odoo Accounting
+
+Manual configuration
+--------------------
+
+If your bank can’t be synchronized automatically, or if you prefer not to sync your bank account
+with your database, you may also configure your bank account manually.
```

Comment:
```suggestion
If your bank can’t be synchronized automatically, or if you prefer not to sync your bank account
with your database, you may also configure your bank account manually.
```

Couldn't you replace the second "Bank Account" with "It" to avoid repeating it two times?

Reply:
Indeed, it's fixed